lib/backend/dynamo/configure.go

```diff
@@ -150,3 +150,44 @@ func getWriteScalingPolicyName(resourceID string) string {
 func getReadScalingPolicyName(resourceID string) string {
 	return fmt.Sprintf("%s-read-target-tracking-scaling-policy", strings.TrimPrefix(resourceID, "table/"))
 }
+
+func TurnOnTimeToLive(ctx context.Context, svc dynamodbiface.DynamoDBAPI, tableName string, ttlKey string) error {
+	status, err := svc.DescribeTimeToLiveWithContext(ctx, &dynamodb.DescribeTimeToLiveInput{
+		TableName: aws.String(tableName),
+	})
+	if err != nil {
+		return convertError(err)
+	}
```

Comment:
LGTM - but why did we remove the `trace.Wrap(...)` around `convertError` here?

Reply:
I've removed it for consistency with the remaining functions in this file (and the `convertError` in the end of this function).
I don't know if there would be any impact in `trace.Wrap`ping all the  `convertError`.